spring-cloud-aws-autoconfigure/pom.xml

```diff
@@ -18,18 +18,22 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-autoconfigure</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>software.amazon.awssdk</groupId>
+			<artifactId>aws-core</artifactId>
```

Comment:
Should this dependency be added to the `spring-cloud-aws-core` module?

Reply:
The autoconfigure module unfortunately depends on it in Config Resolvers.